src/Classes/iTones/iTones_Playlist.php

```diff
@@ -235,6 +235,27 @@ public function getTitle()
         return $this->title;
     }
 
+    public function getPlaylistAtTime($time)
```

Comment:
What will this do if there are multiple active playlists at a time?

Reply:
just return one of them. This shouldn't matter as we are planning to move to only having a single active playlist at any point.